src/data/getUserByEmail.js

```diff
@@ -9,6 +9,14 @@ const debug = require('debug')('appstore:server:data:getUserByEmail')
 const getUserByEmail = async (email, knex) => {
     let user = null
 
+    if (!email) {
+        return null
+    }
+
+    if (!knex) {
+        throw new Error(`Missing parameter 'knex'`)
```

Comment:
We could look into something like https://hapi.dev/family/bounce/?v=1.3.2 to avoid having these everywhere. I guess the reason is due to the catch below which would silently fail due to the 'undefined' error that would otherwise occur, right?

Reply:
will look into bounce!